src/components/pages/pages.stories.mdx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
```

Comment:
we need to ignore `chromatic` here for all stories by passing `{ chromatic: { disable: true }}` into `Meta`

Reply:
Chromatic tests have been disabled too.